<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface_image_correction</class>
 <widget class="QMainWindow" name="interface_image_correction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget_interface_image_correction{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgb(113, 139, 152), stop:0 rgb(40, 75, 92));}\n\n</string>
  </property>
  <widget class="QWidget" name="centralwidget_interface_image_correction">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_dialog_image" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="companovka_dialog_image"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_dialog_controle" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkbox_submit_area">
         <property name="text">
          <string>sub area</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="button_back_submit">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="button_submit_area">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="button_fit">
         <property name="text">
          <string>Fit</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="button_forward_submit">
         <property name="text">
          <string>Forward</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QSlider" name="horizontal_slider_scale">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_image"/>
    <addaction name="action_save_image"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open_image">
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="action_save_image">
   <property name="text">
    <string>Save image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
